components/ml/org.wso2.carbon.ml.core/src/main/java/org/wso2/carbon/ml/core/utils/ComputeClasspath.java

```diff
@@ -0,0 +1,293 @@
+            "org.wso2.carbon.event.admin",
+            "org.wso2.carbon.event.application.deployer",
+            "org.wso2.carbon.event.client",
+            "org.wso2.carbon.event.common",
+            "org.wso2.carbon.event.core",
+            "org.wso2.carbon.event.flow",
+            "org.wso2.carbon.event.input.adapter.core",
+            "org.wso2.carbon.event.input.adapter.email",
+            "org.wso2.carbon.event.input.adapter.filetail",
+            "org.wso2.carbon.event.input.adapter.http",
+            "org.wso2.carbon.event.input.adapter.jms",
+            "org.wso2.carbon.event.input.adapter.kafka",
+            "org.wso2.carbon.event.input.adapter.mqtt",
+            "org.wso2.carbon.event.input.adapter.soap",
+            "org.wso2.carbon.event.input.adapter.websocket.local",
+            "org.wso2.carbon.event.input.adapter.websocket",
+            "org.wso2.carbon.event.input.adapter.wso2event",
+            "org.wso2.carbon.event.output.adapter.cassandra",
+            "org.wso2.carbon.event.output.adapter.core",
+            "org.wso2.carbon.event.output.adapter.email",
+            "org.wso2.carbon.event.output.adapter.http",
+            "org.wso2.carbon.event.output.adapter.jms",
+            "org.wso2.carbon.event.output.adapter.kafka",
+            "org.wso2.carbon.event.output.adapter.logger",
+            "org.wso2.carbon.event.output.adapter.mqtt",
+            "org.wso2.carbon.event.output.adapter.rdbms",
+            "org.wso2.carbon.event.output.adapter.sms",
+            "org.wso2.carbon.event.output.adapter.soap",
+            "org.wso2.carbon.event.output.adapter.websocket.local",
+            "org.wso2.carbon.event.output.adapter.websocket",
+            "org.wso2.carbon.event.output.adapter.wso2event",
+            "org.wso2.carbon.event.processor.common",
+            "org.wso2.carbon.event.processor.core",
+            "org.wso2.carbon.event.processor.manager.commons",
+            "org.wso2.carbon.event.processor.manager.core",
+            "org.wso2.carbon.event.publisher.core",
+            "org.wso2.carbon.event.receiver.core",
+            "org.wso2.carbon.event.simulator.core",
+            "org.wso2.carbon.event.statistics",
+            "org.wso2.carbon.event.stream.core",
+            "org.wso2.carbon.event.tracer",
+            "org.wso2.carbon.utils",
+            "org.spark.project.akka.actor",
+            "org.spark.project.akka.remote",
+            "org.spark.project.akka.slf4j",
+            "config"
+    };
+
+    private static String SEP = System.getProperty("os.name").toLowerCase().contains("win") ? ";" : ":";
+
+    public static void main(String[] args) throws Exception {
+        if (args.length == 0) {
+            throw new Exception("Arguments to the main method should not be empty");
+        }
+
+        String carbonHome = args[0];
+        String sparkClasspath = "";
+
+        System.out.println(getSparkClasspath(sparkClasspath, carbonHome));
```

Comment:
This class was directly copied from carbon-analytics.
However, logging or printing this is not required here. Therefore removing the print statement.